src/_playground/App.scss

```diff
@@ -285,6 +285,10 @@ $menu-spacing: 20px;
     }
 }
 
+.frDocs-Properties__shape {
+    margin: 0 100px 50px 100px;
```

Comment:
This is a pretty dramatic and doesn't seem to work well at 320px (mobile view) for the docs site. Can we drop some of these down while still keeping the indentation. I think the idea of the indentation is great however because it keeps the content looking associated with the above property table but different enough that as these props grow it will stand out in the correct way.